nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -6,17 +6,19 @@ let
   cfg = config.services.nextcloud;
   fpm = config.services.phpfpm.pools.nextcloud;
 
-  phpPackage =
-    let
-      base = pkgs.php74;
-    in
-      base.buildEnv {
-        extensions = { enabled, all }: with all;
-          enabled ++ [
-            apcu redis memcached imagick
-          ];
-        extraConfig = phpOptionsStr;
-      };
+  phpPackage = pkgs.php74.buildEnv {
+    extensions = { enabled, all }:
+      (with all;
+        enabled
+        ++ [ imagick ] # Always enabled
+        # Optionally enabled depending on caching settings
+        ++ optional cfg.caching.apcu apcu
+        ++ optional cfg.caching.redis redis
+        ++ optional cfg.caching.memached memcached
```

Comment:
This doesn't evaluate since `memached` doesn't exist and it's supposed to be `memcached`.

In case you didn't know, you can run the tests on your own `nix-build nixos/tests/nextcloud/basic.nix --option builders '' ` :)

Reply:
Thanks for catching! Fixed. And this time, I ran the test before pushing.